source/agora/consensus/EnrollmentManager.d

```diff
@@ -1300,44 +1248,45 @@ unittest
 
     // create and add the first enrollment
     Enrollment[] enrolls;
-    auto enroll = man.createEnrollment(utxo_set.keys[0]);
+    auto enroll = man.createEnrollment(utxo_set.keys[0], Height(10));
     assert(man.addEnrollment(enroll, key_pair.address, Height(10),
             utxo_set.getUTXOFinder()));
 
     // if the current height is smaller than the available height,
     // we can get no enrollment
-    man.getEnrollments(enrolls, Height(9));
+    man.getEnrollments(enrolls, Height(9), &utxo_set.peekUTXO);
     assert(enrolls.length == 0);
```

Comment:
> Ledger calls getEnrollments with the current height, EnrollmenManager request the enrollments valid for next block from the EnrollmentPool

Ledger calls getEnrollments with the current height(n-1 = 9), EnrollmenManager request the enrollments valid for next block(n = 10) from the EnrollmentPool.
This is what I meant.